books/free-programming-books-subjects.md

```diff
@@ -299,10 +299,12 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 ### Graphics Programming
 
 * [3D Game Shaders For Beginners](https://github.com/lettier/3d-game-shaders-for-beginners) - David Lettier (Git) [(HTML)](https://lettier.github.io/3d-game-shaders-for-beginners)
+* [Augmented Reality](https://www.researchgate.net/publication/227164365_Augmented_Reality_An_Overview) - Julie Carmigniani and Borko Furht
```

Comment:
Don't invent titles! Use the title given by the resource.
This is a book chapter, not a book. We don't list book chapters.